src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -252,3 +255,137 @@ def test_WINNF_FT_S_SCS_15(self,config_filename):
 
     # Check registration response
     self.assertEqual(response['response']['responseCode'],104)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_17(self):
+    """Invalid certificate following an approved registration request
+       Checks that SAS UUT response with fatal alert message.
```

Comment:
This 3-space indentation is inconsistent with the other docstrings.  In general, the second line should be blank:
https://www.python.org/dev/peps/pep-0008/#doc

(This applies throughout the pull request.)

Reply:
Updated   accordingly